exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,28 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+
+  if (number === 5){
+    return false
+  }
+if (number < 0){
```

Comment:
This line 14 is correct 😄 
Can you change line 11 to work for all other positive numbers, not just 5?